app/src/main/res/layout/favorites_layout.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical" android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Favorites"
+        android:padding="8dp"
+        android:textColor="#fff"
+        android:textSize="28sp"
+        android:id="@+id/textView"
```

Comment:
Use descriptive names for your ids. it makes it less painful in debugging and maintainance, I promise :smile: